android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -63,17 +58,17 @@ class FlutterInboxMessageView(private var context: Context) : PlatformView, Meth
         view.webViewClient = object: UAWebViewClient() {
             override fun onPageStarted(view: WebView, url: String?, favicon: Bitmap?) {
                 super.onPageStarted(view, url, favicon)
-                EventManager.shared.notifyEvent(WebviewLoadStartedEvent())
+                channel.invokeMethod("onLoadStarted", null)
             }
 
             override fun onPageFinished(view: WebView?, url: String?) {
                 super.onPageFinished(view, url)
-                EventManager.shared.notifyEvent(WebviewLoadFinishedEvent())
+                channel.invokeMethod("onLoadFinished", null)
             }
 
             override fun onClose(webView: WebView) {
                 super.onClose(webView)
-                EventManager.shared.notifyEvent(WebviewClosedEvent())
+                channel.invokeMethod("onClose", null)
```

Comment:
Where is onLoadError being sent?

Reply:
in onReceivedError through FlutterResult